parsec/api/data/manifest.py

```diff
@@ -190,14 +198,14 @@ def make_obj(self, data):
 
     id: EntryID
     parent: EntryID
-    version: int
```

Comment:
and version should also be remove from SCHEMA_CLS !

Reply:
I don't think so, there is no inheritance between `Manifest.SCHEMA_CLS` class and the Manifest subclasses `SCHEMA_CLS` vlass